modules/client/examples/01-client/13-prepare-installation.ts

```diff
@@ -0,0 +1,72 @@
+/* MARKDOWN
+---
+title: PrepareInstallation
+---
+### Prepare installation of a plugin
+*/
+
+import {
+  Client,
+  PrepareInstallationParams,
+  PrepareInstallationStep,
+} from "@aragon/sdk-client";
+import { context } from "../index";
+
+// Instantiate the general purpose client from the Aragon OSx SDK context.
+const client: Client = new Client(context);
+
+const prepareInstallationParams: PrepareInstallationParams = {
+  daoAddressOrEns: "0x1234567890123456789012345678901234567890", // my-dao.dao.eth
+  pluginRepo: "0x2345678901234567890123456789012345678901",
+  installationParams: [ // Parameters needed by the prepare install abi
+    1234,
+    "0x1234567890123456789012345678901234567890",
+  ],
+  installationAbi: ["uint256", "adress"],
```

Comment:
the problem with that abi is that is not compatible with `ParamType`which is accepted by the `defaultAbiEncoder`, so either we accept it as a string or we accept it with the metadata format and we parse it internally to the string format

```ts
export class ParamType {

    // The local name of the parameter (of null if unbound)
    readonly name: string;

    // The fully qualified type (e.g. "address", "tuple(address)", "uint256[3][]"
    readonly type: string;

    // The base type (e.g. "address", "tuple", "array")
    readonly baseType: string;

    // Indexable Paramters ONLY (otherwise null)
    readonly indexed: boolean;

    // Tuples ONLY: (otherwise null)
    //  - sub-components
    readonly components: Array<ParamType>;

    // Arrays ONLY: (otherwise null)
    //  - length of the array (-1 for dynamic length)
    //  - child type
    readonly arrayLength: number;
    readonly arrayChildren: ParamType;

    readonly _isParamType: boolean;
```